### DISCUSSION TITLE
Updating Heimdall failed

### DISCUSSION THREAD

**Discussion by HolgerM1** [tteck/Proxmox#3159] [Help Needed]
Hi,
I was trying to update Heimdall from 2.5 (something) to 2.6.1. Unfortunately, the update failed. After reboot, Heimdall gui is no longer reachable. Any hints here?

Thanks!
![Screenshot 2024-06-09 155510](https://github.com/tteck/Proxmox/assets/114694650/43d08c07-9293-483c-942b-1a78a29d7c56)

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/2772

**Reply by HolgerM1** [reply]
Thanks, could have found that myself, sorry.

**Reply by HolgerM1** [reply]
I have to come back to that. I upgraded to Debian 12 #1498 and then updated Heimdall again. Still the same error message?!
![Screenshot 2024-06-09 165033](https://github.com/tteck/Proxmox/assets/114694650/58c9784d-8743-4317-a60c-c979f3295433)
![Screenshot 2024-06-09 170239](https://github.com/tteck/Proxmox/assets/114694650/f23e7c20-d1d6-4510-9794-8c6b513653fa)
